actions/check_is_bot_in_voice_channel.js

```diff
@@ -100,7 +100,13 @@ module.exports = {
 	action(cache) {
 		const data = cache.actions[cache.index];
 		const Audio = this.getDBM().Audio;
-		const result = cache.server ? Audio.getSubscription(cache.server) : false;
+		let result = false;
+		if (cache.server) {
+			result = Audio.getSubscription(cache.server);
+			if (!result) {
+				result = !!cache.server.members?.me?.voice?.channel;
```

Comment:
No reason to invoke the `channel` getter

```suggestion
				result = !!cache.server.members?.me?.voice?.channelId;
```